org.gnome.Evince.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.gnome.Evince",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.32",
+    "runtime-version": "3.34",
```

Comment:
Which version of cairo is available at 3.34 runtime? If anything higher than 1.16, then we either should stick to runtime 3.32 or build cairo ourselves (if master, reverting a commit as explained in https://gitlab.freedesktop.org/cairo/cairo/issues/376)

Reply:
I think it ships `1.16.0-0-g3ad43122b21a3299dd729dc8462d6b8f7f01142d` as per https://gitlab.com/freedesktop-sdk/freedesktop-sdk/blob/19.08/elements/components/cairo.bst#L46

Might be a good time to implement an (upstream) test case for this, no?